app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -265,6 +266,25 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
                 }
 
                 override fun onPageFinished(view: WebView?, url: String?) {
+                    // Use idea from https://github.com/home-assistant/iOS/pull/1472 to filter viewport
```

Comment:
Should we also reset the values like iOS does when the feature is turned off?

Reply:
Yes I think we should.
I'll work on it tonight

Reply:
Please review & check latest version